src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -3,6 +3,7 @@
 /** @jsx h */
 import { h, Fragment } from 'preact';
 import Instructions from '../../Instructions';
+import Disclaimer from '../../Disclaimer';
 import Donut from '../../Donut';
 import ProductListLink from '../../ProductListLink';
 import InlineLinks from '../../InlineLinks';
```

Comment:
```suggestion
import Disclaimer from '../../Disclaimer';
import Donut from '../../Donut';
import ProductListLink from '../../ProductListLink';
import InlineLinks from '../../InlineLinks';
import Button from '../../Button';
```
Let's import the `Button` component since we'll need that to show the checkout button at the bottom.